static/js/pm_pill.js

```diff
@@ -0,0 +1,61 @@
+import * as input_pill from "./input_pill";
+import * as people from "./people";
+import * as user_pill from "./user_pill";
+import * as util from "./util";
+
+export class PmPill {
```

Comment:
Can you add some comment documentation here on how this differs from the `user_pill` module? It's not clear to me what the distinction is, and I'm also curious whether it'd make sense to make the special behavior here just a parameter that can be passed when initializing a user pill vs. whether it's appropriate to do a separate class.

Reply:
Finally, it might make sense to make this commit earlier in the branch -- in theory, this is a refactor, and it might be preferable to integrate it before the modal is introduced in what is currently the previous commit.

Reply:
> Can you add some comment documentation here on how this differs from the user_pill module?

This new class just uses `user_pill` as a utils module -- it's not something we can initialise. `PmPill` is mainly a wrapper around `input_pill`, an instance of which it passes to `user_pill` functions for processing for most of the `PmPill` functions... Does that answer your question? (I'm not sure I understood it correctly 😅)